cmd/utils/utils.go

```diff
@@ -20,6 +21,43 @@ func EnsureProjectDir(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func EnsurePreFlightSetup(cmd *cobra.Command, args []string) error {
+	if err := EnsureProjectDir(cmd, args); err != nil {
+		return err
+	}
+
+	if config.CFG.DockerCommand.GetString() == "podman" {
+		if err := airflow.InitPodmanMachineCMD(); err != nil {
+			return err
+		}
+	}
```

Comment:
Once we merge this PR, we can use the new function, so it'd be automatic when we get everything integrated. https://github.com/astronomer/astro-cli/pull/1738